browser/ui/views/profiles/brave_profile_menu_view_helper.cc

```diff
@@ -8,14 +8,17 @@
 #include "brave/browser/brave_browser_process_impl.h"
 #include "brave/browser/profiles/profile_util.h"
 #include "brave/browser/tor/buildflags.h"
+#include "brave/common/pref_names.h"
 #include "brave/grit/brave_generated_resources.h"
 #include "chrome/app/vector_icons/vector_icons.h"
+#include "components/prefs/pref_service.h"
```

Comment:
why was this added?

Reply:
Deleted. It was used directly at first but replaced with `tor::TorProfileService::IsTorDisabled()`